docs/b9lab-processes.md

```diff
@@ -0,0 +1,93 @@
+# B9lab processes
+
+This file describes the internal processes for the [B9lab](https://b9lab.com) contribution team. External contributors should follow the [Contribution Guide](../CONTRIBUTING.md).
+
+
+## General
+
+**We are exclusively working on the public github repository at [https://github.com/cosmos/sdk-tutorials](https://github.com/cosmos/sdk-tutorials)!**
+
+Our default branch naming scheme applies: A branch should be named `xx-branch-description`, there xx should be replaced with your initials (lowercase), and the branch description should give enough context to understand what is in this branch. For example, this file has been updated through a branch named `is-update-guidelines`.
+
+
+## Creating a new Pull Request
+
+When creating a new Pull Request, you **must** follow these steps:
+
+1. Select the `b9lab_pr_template` as Pull request template.
+2. Set a [descriptive name](https://cbea.ms/git-commit/), including a category prefix if applicable:
```

Comment:
@coldice I would recommend adding an example for the prefixes to make it clearer.